package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "bookstore-react",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.5.1",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "prop-types": "^15.7.2",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-redux": "^7.2.3",
```

Comment:
- [ ] [REQUIRED] Good job by adding react-redux but, it will be nice if you add also `redux` to `package.json ` [screenshot from Pathwright - redux](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone1/AddReduxAndReact-Redux.png),  as you will need it in the next milestone.

Reply:
@AkintoyeOlamide For some reason. you forgot to attend to the issue mentioned by the first reviewer, let's try to always read the comment from the reviewer carefully, doing this will reduce the number of CRR.

@AkintoyeOlamide kindly add `redux` to `package.json`
![image](https://user-images.githubusercontent.com/31897434/127852307-a6758d1b-0f49-4a33-9237-9d9d7a477569.png)